### PR TITLE
DOCS-605: Description for the always_fetch_merged_part setting

### LINKED ISSUE #11754: SQL: GRANT ALL *.* in wrong order
**Describe the issue**
On file, `/docs/en/operations/access-rights.md` under usage section,  the SQL is given as 
`GRANT ALL ON *.* WITH GRANT OPTION TO admin_user_account`. It should be `GRANT ALL ON *.*  TO admin_user_account WITH GRANT OPTION`
**Additional context**
..

### PR DESCRIPTION
**Pull request by BayoNet**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:
- Documentation for #10379

Description
- Described always_fetch_merged_part
- Added Plausible to adopters
- Fixed #11754

### CODE REVIEW

File: docs/en/introduction/adopters.md [hunk 1]
```diff
@@ -50,7 +50,8 @@ toc_title: Adopters
...
+| <a href="https://www.percent.cn/" class="favicon">Percent 百分点</a>                            | Analytics                       | Main Product          | —                                                          | —                                                                            | [Slides in Chinese, June 2019](https://github.com/ClickHouse/clickhouse-presentations/blob/master/meetup24/4.%20ClickHouse万亿数据双中心的设计与实践%20.pdf)                                                            |
+| <a href="https://plausible.io/" class="favicon">Plausible</a>                                  | Analytics                       | Main Product          | —                                                          | —                                                                            | [Blog post, June 2020](https://twitter.com/PlausibleHQ/status/1273889629087969280)                                                                                                                                      |
```
Comment (blinkov) [inline docs/en/introduction/adopters.md:54]: twitter contained a link to an actual blog post, no need for extra indirection

File: docs/en/introduction/adopters.md [hunk 2]
```diff
@@ -50,7 +50,8 @@ toc_title: Adopters
...
-| <a href="https://www.percent.cn/" class="favicon">Percent 百分点</a>                           | Analytics                       | Main Product          | —                                                          | —                                                                            | [Slides in Chinese, June 2019](https://github.com/ClickHouse/clickhouse-presentations/blob/master/meetup24/4.%20ClickHouse万亿数据双中心的设计与实践%20.pdf)                                                            |
+| <a href="https://www.percent.cn/" class="favicon">Percent 百分点</a>                            | Analytics                       | Main Product          | —                                                          | —                                                                            | [Slides in Chinese, June 2019](https://github.com/ClickHouse/clickhouse-presentations/blob/master/meetup24/4.%20ClickHouse万亿数据双中心的设计与实践%20.pdf)                                                            |
```
Comment (alesapin) [inline docs/en/introduction/adopters.md:53]: Not related to changes?

File: docs/en/operations/settings/settings.md [hunk 3]
```diff
@@ -1418,6 +1418,23 @@ Possible values:
 
 Default value: 16.
 
+## always_fetch_merged_part {#always_fetch_merged_part}
+
+Prohibits data parts merging at replicas in [Replicated*MergeTree](../../engines/table-engines/mergetree-family/replication.md)-engine tables.
```
Comment (alesapin) [inline docs/en/operations/settings/settings.md:1423]: Don't need `at replicas`. Just:
```suggestion
Prohibits data parts merging in [Replicated*MergeTree](../../engines/table-engines/mergetree-family/replication.md)-engine tables.
```

File: docs/ru/operations/settings/settings.md [hunk 4]
```diff
@@ -1205,6 +1205,23 @@ Default value: 0.
 
 Значение по умолчанию: 16.
 
+## always_fetch_merged_part {#always_fetch_merged_part}
+
+Запрещает слияние данных на репликах для таблиц семейства [Replicated*MergeTree](../../engines/table-engines/mergetree-family/replication.md).
```
Comment (alesapin) [inline docs/ru/operations/settings/settings.md:1210]:
```suggestion
Запрещает слияние данных для таблиц семейства [Replicated*MergeTree](../../engines/table-engines/mergetree-family/replication.md).
```

File: docs/en/introduction/adopters.md (hunk 2 above)
Comment (BayoNet) [reply docs/en/introduction/adopters.md:53]: Yes.

File: docs/en/introduction/adopters.md (hunk 1 above)
Comment (BayoNet) [reply docs/en/introduction/adopters.md:54]: It looks like twitter contain more valuable information, than the blog post itself.

**Pull request merged**